### ISSUE TITLE
SVG and Microsoft Edge 15

### ISSUE THREAD

**Issue by erthi16** [kenwheeler/slick#3010]
Hello to all!

I had bugs with SVG images inside my slider when using Microsoft Edge and issue 2371 solved my problem by reapplying the data to each SVG in the Slick init function.

Onfortunately, my boss just received his new PC and all my SVG images have dissapeared again in Microsoft Edge!

The interesting thing is that his version of Edge differes from mine and all other that I've tested on! I have Edge 14 and the buggy version is 15!!!!!

The CanIUse site specifies that transform 3D is supported by all version of Edge but not transorm 2D...
[http://caniuse.com/#search=transform](http://caniuse.com/#search=transform)

Here are 2 tests pages:

my fiddle but no Edge version work... strange: [http://jsfiddle.net/xg7escL2/](http://jsfiddle.net/xg7escL2/)
test page in my site: Edge 14 is OK but not Edge 15 [https://www.ulaval.ca/index.php?id=37615](https://www.ulaval.ca/index.php?id=37615)

To reproduce the problem:

1. With Microsoft Edge < 15 
2. Empty cach
3. Reload page with CTRL+F5
4. The 2 logos are present

5. With Microsoft Edge 15+
6. Empty cach
7. Relead page with CTRL+F5
8. No logos

So there is something that the new version of Edge does not like in the animation / transition because sometimes I see the SVG while sliding but dessapear after transition.  

So the new version of Microsoft Edge (15+) is buggy with Slick
JQuery 3.2.1
Slick 1.6 and 1.5.9

Thank you for your help!

**Comment by erthi16** [comment]
I've made great progress...

I've pasted the code of every SVG directly in my page (huge mess) and... mirracle! Microsoft Edge 15 is behaving! I also had to use [https://codepen.io/digitalmouse/pen/LygOWe](https://codepen.io/digitalmouse/pen/LygOWe) solution to reactivate scalling in Internet Explorer 11.

So it seems that Edge 15 is no longer able or willing to interprete what is embeded inside a SVG file when transition 3D or 2D are used!
